### PR TITLE
update 1.25.3-dev changelog

### PR DESCRIPTION
**Pull request by suneyz**
### Summary

Add change log 1.25.3 release

### Testing

New tests cover the changes: 

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.25.3-dev
+* Bug - Update router to clean url with multi-slash, it will redirect "//path///to/handler" to "/path/to/handler" [#1844](https://github.com/aws/amazon-ecs-agent/pull/1844)
```
Comment (sharanyad) [inline CHANGELOG.md]: I would prefer having it more descriptive for our customers. 
eg: `Fixed a bug where agent no longer redirected credentials URL` or something similar..

Comment (yunhee-l) [inline CHANGELOG.md]: can you link the issue?

Comment (suneyz) [reply CHANGELOG.md]: updated:)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.25.3-dev
+* Bug - Fixed a bug where agent no longer redirected matadata query with multi-slash url to correct url [#1839](https://github.com/aws/amazon-ecs-agent/issues/1839) [#1844](https://github.com/aws/amazon-ecs-agent/pull/1844)
```
Comment (sharanyad) [inline CHANGELOG.md]: we generally don't link issues in changelog.
also nit: `metadata`

Comment (suneyz) [reply CHANGELOG.md]: updated

**Pull request merged**